### DISCUSSION TITLE
How to create a time filter which filter less OR equal than

### DISCUSSION THREAD

**Discussion by Razikus** [apache/superset#27206] [Q&A / Help]
I cannot create it 
![Screenshot from 2024-02-22 17-01-58](https://github.com/apache/superset/assets/12664801/ab498e28-1db3-48fe-9e14-97cf9e1b7a32)

So basically because i have data that is created today, it will not show any data

**Comment by cyzchenyuzhu** [comment]
同问

**Comment by Razikus** [comment]
bump

**Comment by michael-s-molina** [comment]
<img width="589" alt="Screenshot 2024-03-11 at 08 51 35" src="https://github.com/apache/superset/assets/70410625/93aeac18-3278-47c6-b8f6-c1189ca991d4">
